SDAPackage/SDA/obstacle_map.py

```diff
@@ -133,6 +133,7 @@ def is_obstacle_in_path(self):
                 if isinstance(obstacle, StationaryObstacle):
                     paths = self.generate_possible_paths(obstacle)
                     if len(paths) != 0:
+                        print(len(paths))
                         return True, np.array(paths)
                 #elif isinstance(obstacle, MovingObstacle):
                 #    pass
```

Comment:
We have removed moving obstacles from this iteration of SDA, so please remove the commented out code here.

Reply:
done